bitnami/harbor/templates/registry/registry-svc.yaml

```diff
@@ -12,8 +13,8 @@ metadata:
 spec:
   ports:
     - name: registry
-      port: {{ include "harbor.registry.servicePort" . }}
+      port: {{ ternary .Values.registry.server.containerPorts.https .Values.registry.server.containerPorts.http .Values.internalTLS.enabled }}
```

Comment:
Same, this should be service port, not container port

Reply:
To be resolved in previous conversation.